homework.md

```diff
@@ -4,77 +4,70 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+CSS is used to style websites. 
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
-```css
-#h1 {
-
-}
-```
+n#id h1
```

Comment:
I don't think this is correct. Say if you want to change the style for the h1 tag, how would you do it in CSS?

Reply:
I have only just realised that the original answer to this question is correct. Therefore didn't need correction. Thanks

Reply:
No the original answer is not correct. Check again 😉